packages/ui/src/components/va-file-upload/VaFileUploadUndo/VaFileUploadUndo.vue

```diff
@@ -58,7 +58,7 @@ export default defineComponent({
       vertical: props.vertical,
     }))
 
-    const undoDurationStyle = computed(() => `${undoDuration.value}ms`)
+    const undoDurationStyle = computed(() => `${undoDuration?.value}ms`)
```

Comment:
Is `undefined` converts to `0` in styles block or throws warning? If you not sure about this, let's make: `${undoDuration?.value ?? 0}ms`

Reply:
Doesn't it will replace with default `inject` value from above?

Reply:
Yes, it will, but `undefined` can also be provided-injected, overwriting default value. For some reasons you changed to `?.`, so there are some ways for `undoDuration.value` to be `undefined`.